libs/fgdbg/include/fgdbg/Session.h

```diff
@@ -42,16 +43,24 @@ class Session {
     /**
      * Send session information to server
      */
-    void update() const;
+    void update();
 
 private:
     const std::string& name;
```

Comment:
that looks dangerous. A reference is just like a pointer in C++ there is no guarantee about the lifetime of the object. I would just use a `std::string` without `&`

Reply:
good catch, thanks!